jcl/src/java.base/share/classes/com/ibm/oti/vm/VM.java

```diff
@@ -583,4 +585,17 @@ public static int markCurrentThreadAsSystem()
  * @return true if JVM is in single threaded mode, false otherwise
  */
 public static native boolean isJVMInSingleThreadedMode();
+
+public static ConstantPool getConstantPoolFromAnnotationBytes(byte[] array) {
+	long offset = Unsafe.ARRAY_BYTE_BASE_OFFSET + ((array.length * Unsafe.ARRAY_BYTE_INDEX_SCALE) - FJ9OBJECT_SIZE);
```

Comment:
The constant pool is appended to MethodAnnotationData, MethodParametersAnnotation and MethodDefaultAnnotationData by this change. You may want to make a comment on the passed in parameter `array` that it need to be these data array.

I guess some exiting calls on the class annotation data goes through the same operation (fetching internalCP from the data array and convert to `ConstantPool`). They may be able to reuse this function. But it does not need to be done in this PR.